### PR TITLE
Remove duplicated flag

### PR DESCRIPTION
**Pull request by StrongMonkey**

### CODE REVIEW

**Comment by StrongMonkey** [comment]
https://github.com/rancher/fleet/issues/336

File: modules/cli/cmds/test.go [hunk 1]
```diff
@@ -19,7 +19,7 @@ type Test struct {
 	BundleInputArgs
 	Quiet       bool              `usage:"Just print the match and don't print the resources" short:"q"`
 	Group       string            `usage:"Cluster group to match against" short:"g"`
-	ClusterName string            `usage:"Cluster name to match against" short:"n"`
+	ClusterName string            `usage:"Cluster name to match against"`
```
Comment (ron1) [inline modules/cli/cmds/test.go:22]: I suspect ClusterName will be a popular option. What would you think about using capital 'N' as a short?

**Review by nickgerace — APPROVED** [review]

File: modules/cli/cmds/test.go (hunk 1 above)
Comment (StrongMonkey) [reply modules/cli/cmds/test.go:22]: I am not sure if Capital flag is good idea here. @nickgerace Thoughts?

Comment (nickgerace) [reply modules/cli/cmds/test.go:22]: We could use `-c`, but I agree that capital flags are uncommon enough that we should only use them when necessary.

Comment (StrongMonkey) [reply modules/cli/cmds/test.go:22]: It looks like we already have `-L`. To make it consistent I will rename to it `name` and shortflag it as `-N`

**Review by nickgerace — APPROVED** [review]

**Pull request merged**